schema.graphql

```diff
@@ -230,6 +263,10 @@ input AddOpticalImageInput {
 
 type Query {
   dataset(id: String!): Dataset
+  user(id: String!): UserProfile
+  group(id: String!): Group
```

Comment:
@andy-d-palmer just to clarify, should this return null if the requester is not in the group?

Reply:
yes